src/util/regex.ts

```diff
@@ -9,5 +13,6 @@ export const testCljOrJsRegex = (regexp: string, str: string) => {
   const clojureReMatches = regexp.match(/^#"(.*)"$/);
   const normalizedRe =
     (clojureReMatches && RegExp(clojureReMatches[1])) || RegExp(trim(regexp, '/'));
+  console.log(normalizedRe, str, normalizedRe.test(str.replace(/^.*\//, '')));
```

Comment:
Did you mean to leave this log statement in the code?

Reply:
Not intended. I'll fix. Thanks!